ee/fe/modules/340-monitoring-applications/applications/rabbitmq/grafana-dashboards/rabbitmq.json

```diff
@@ -219,71 +245,75 @@
         "y": 0
       },
       "id": 66,
-      "interval": null,
       "links": [],
       "maxDataPoints": 100,
       "options": {
         "colorMode": "background",
-        "fieldOptions": {
-          "calcs": ["lastNotNull"]
-        },
         "graphMode": "area",
         "justifyMode": "auto",
         "orientation": "horizontal",
         "reduceOptions": {
-          "calcs": ["mean"],
+          "calcs": [
+            "lastNotNull"
+          ],
           "fields": "",
           "values": false
-        }
+        },
+        "textMode": "auto"
       },
-      "pluginVersion": "7.0.6",
+      "pluginVersion": "8.5.13",
       "targets": [
         {
-          "expr": "count(rabbitmq_channel_messages_published_total * on(instance) group_left(rabbitmq_cluster) rabbitmq_identity_info{namespace=\"$namespace\",job=\"rabbitmq\"})",
```

Comment:
In this case, we need to use 'or' for backwards compatibility with older versions. This also applies to all subsequent changes.